.github/workflows/release-notifier.yml

```
@@ -0,0 +1,20 @@
+name: Release Notifier
+on: release
+
+jobs:
+  notify:
+    name: release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - run: npm ci
```

I just realized this won't work unless we add a `package.json` and `package-lock.json` to the project.

As a simple workaround, we could replace this with

```suggestion
      - run: npm install @octokit/auth-app @octokit/core @octokit/plugin-paginate-rest
```

The more elegant solution will be to encapsulate the code from `release-notifier` into a reusable action itself, something I planned on doing for a long time now anyway, can as well do it now